Styles.css

```diff
@@ -0,0 +1,323 @@
+  text-decoration: none;
+  display: flex;
+  align-items: center;
+}
+
+.navLink:hover {
+  background: darkgray;
+  color: black;
+}
+
+button {
+  padding-left: 22px;
+  padding-right: 22px;
+  padding-top: 12px;
+  padding-bottom: 12px;
+  background: black;
+  font-size: 1.05rem;
+  color: #fff;
+  border-style: none;
+}
+
+button:hover{
+  background: darkgray;
+  color: black;
+}
+
+button:active{
+  background-color: gray;
+}
+
+.concertPicture {
+  position: relative;
+  text-align: center;
+  width: 100%;
+  height: auto;
+}
+
+.wallpaperDesc {
+  color: rgb(255, 255, 255);
+  position: absolute;
+  bottom: 12px;
+  left: 50%;
+  transform: translate(-50%, 0%);
+}
+
+.section {
+  padding-top: 45px;
+}
+
+
+img {
+  z-index: 1;
+  width: 100%;
+  height: auto;
+}
+
+.concertCityImage {
+  max-height: 270px;
+}
+
+.aboutBand {
+  position: relative;
+  text-align: center;
+  background-color: white;
+  margin-bottom: 45px;
+  width: 100%;
+}
+
+.bandName {
+  font-size: 2rem;
+  text-align: center;
+}
+
+.descContainer {
+  align-items: center;
+  font-size: 1.05rem;
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+}
+
+.description {
+  max-width: 800px;
+}
+
+.bandMembers {
+  display: flex;
+  justify-content: space-around;
+  width: 100%;
+  max-width: 800px;
+  margin-top: 52px;
+}
+
+.bandMemberPicture {
+  display: flex;
+  max-width: 105px;
+  flex-direction: column;
+}
+
+.tourTickets  {
+  width: 100%;
+  background-color: black;
+  color: white;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.title {
+  font-size: 2.25rem;
+  text-align: center;
+  width: 100%;
+  margin-top: 22px;
+  margin-bottom: 22px;
+}
+
+.rowDiv {
+  margin-bottom: 5px;
+  padding-left: 5px;
+  padding-right: 5px;
+  color: black;
+  background-color: white;
+  display: flex;
+}
+
+.avilableTickets {
+  padding: 4px;
+  margin: 4px 2px;
+  background-color: black;
+  color: #fff;
+  border-radius: 50%;
+}
+
+.sold {
+  background-color: red;
+  color: #fff;
+  margin-left: 15px;
+}
+
+.ticketsAvilable {
+  margin-top: 4px;
+  margin-bottom: 4px;
+  display: flex;
+  justify-content: space-between;
+  width: 100%;
+}
+
+.ticketsSoldOut {
+  display: flex;
+  margin-top: 4px;
+  margin-bottom: 4px;
+}
+
+.concertCity {
+  max-width: 225px;
+  margin-top: 22px;
+  margin-left: 22px;
+  margin-bottom: 37px;
+  margin-right: 22px;
+  background-color: #fff;
+  color: black;
+}
+
+.buyTicketsText {
+  font-size: 1.05rem;
+  margin-bottom: 22px;
+}
+
+.ticketRows {
+  margin-left: 22px;
+  margin-right: 22px;
+}
+
+.concertDesc {
+  margin-bottom: 12px;
+  margin-right: 12px;
+  margin-left: 12px;
+}
+
+.concertCitys {
+  display: flex;
+  align-items: center;
+  justify-content: space-around;
+}
+
+.nameAndEmail {
+  margin-bottom: 6px;
+  width: auto;
+  display: flex;
+  flex-direction: row;
+}
+
+.contactRight {
+  right: initial;
+  max-width: 400px;
+  width: 50%;
+  min-width: 225px;
```

Comment:
can you use one of this properties?

Reply:
No. Because the width of this div must be betwin 225px and 400px  width the half length of it's container

Reply:
then why it's width is 50%?